addons/ttf/ttf.c

```diff
@@ -1005,10 +1005,10 @@ static bool ttf_get_glyph_dimensions(ALLEGRO_FONT const *f,
       }
    }
    cache_glyph(data, face, ft_index, glyph, false);
-   *bbx = glyph->offset_x;
+   *bbx = glyph->offset_x + 1;
    *bbw = glyph->region.w - 2;
```

Comment:
Like I said, the problem is not here but in the example (ex_ttf) as far I I know. When you draw a bounding box you always draw x + w - 1 not x + w (think about it for a moment.) If you look closely at ex_ttf, the x position is right but the box is drawn 1 pixel too wide and tall.